### DISCUSSION TITLE
Live WordPress website > Local Debug in Ray

### DISCUSSION THREAD

**Discussion by LauGau** [spatie/ray#321] [Q&A] [answered]
I was dreaming about being able to debug my live WordPress websites through Ray locally but I guess it's impossible right now.

My question are :   
1. Is debugging locally in Ray a live WordPress website planned ?
2. Is there any workaround for now or alternatives (even if it's an online debugging tool) ?

Thank you 🙏

**Comment by freekmurze** [comment, marked as answer]
We do have a wordpress specific package: https://github.com/spatie/wordpress-ray

In the next few weeks we'll make it possible to easily connect to external servers in the Ray app!

**Reply by LauGau** [reply]
Woops, sorry I posted on the wrong repo... and thanks for the superquick answer.
That's a great news ! Count on me to take a licence when it will be available !
I cannot wait :)

**Reply by freekmurze** [reply]
Keep on eye on my twitter account 🙂 

We might already launch this early next week 😬
